nntrainer/models/neuralnet.h

```diff
@@ -100,7 +100,8 @@ class NeuralNetwork : public ml::train::Model {
     compiled(false),
     def_name_count(0),
     loadedFromConfig(false),
-    app_context(app_context_) {}
+    app_context(app_context_),
+    in_place_bn_layer_optimization(true) {}
```

Comment:
How about we add a default argument for the option?

Reply:
added.